### PR TITLE
Jnpeterson ce

### PR DESCRIPTION
**Pull request by jeffreynpeterson**
Initial CE setup for Swinburne CE

### CODE REVIEW

File: topology/Swinburne University of Technology/FACILITY.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+ID:
```
Comment (matyasselmeci) [inline topology/Swinburne University of Technology/FACILITY.yaml:1]:
```suggestion
ID: 10182
```

File: topology/Swinburne University of Technology/SUT - OzStar/SITE.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# LongName is the expanded name of the site
+LongName: Swinburne University of Technology
+# Description is a brief description of your site
+Description: OzStar Cluster
+
+# If you have an up-to-date local git clone, fill ID with the output from `bin/next_site_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+ID:
```
Comment (matyasselmeci) [inline topology/Swinburne University of Technology/SUT - OzStar/SITE.yaml:8]:
```suggestion
ID: 10287
```

File: topology/Swinburne University of Technology/SUT - OzStar/SUT-OzStar.yaml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+Production: true
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: Self Supported
+
+# GroupDescription is a long description of the resource group; may be multiple lines.
+GroupDescription: This is a LIGO cluster in Melburne Australia
+
+# If you have an up-to-date local git clone, fill GroupID with the output from `bin/next_resource_group_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID:
```
Comment (matyasselmeci) [inline topology/Swinburne University of Technology/SUT - OzStar/SUT-OzStar.yaml:11]:
```suggestion
GroupID: 1101
```

File: topology/Swinburne University of Technology/SUT - OzStar/SUT-OzStar.yaml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  # Resource Names need to be unique across all resources in the OSG.
+  SUT-OzStar:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: true
+    # Description is a long description of the resource; may be multiple lines
+    Description: This is a Hosted CE for Swinburne University of Technology.
+    # If you have an up-to-date local git clone, fill ID with the output from `bin/next_resource_id`
+    # Otherwise, leave it blank and we will fill in the appropriate value for you.
+    ID:
```
Comment (matyasselmeci) [inline topology/Swinburne University of Technology/SUT - OzStar/SUT-OzStar.yaml:27]:
```suggestion
    ID: 1116
```

**Review by matyasselmeci — CHANGES_REQUESTED** [review]
I filled in the IDs as suggestions.

File: topology/Swinburne University of Technology/SUT - OzStar/SUT-OzStar.yaml [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+        #   ### endpoint (for perfSONAR services)
+        #   # endpoint: <HOSTNAME>
+
+      # Other services if you have any
+      # <SERVICE NAME>:
+      # ...
+
+    ### Tags (optional) is a list of tags associated with the resource.
+    ### Include the tag "CC*" if applicable for a CC* CE.
+    Tags:
```
Comment (brianhlin) [inline topology/Swinburne University of Technology/SUT - OzStar/SUT-OzStar.yaml:94]:
```suggestion
    # Tags:
```

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**